sound/soc/intel/boards/sof_realtek_common.c

```diff
@@ -241,3 +242,118 @@ void sof_rt1015p_codec_conf(struct snd_soc_card *card)
 	card->codec_conf = rt1015p_codec_confs;
 	card->num_configs = ARRAY_SIZE(rt1015p_codec_confs);
 }
+
+/*
+ * RT1015 audio amplifier
+ */
+static const struct snd_soc_dapm_route speaker_map_lr[] = {
+	{ "Left Spk", NULL, "Left SPO" },
+	{ "Right Spk", NULL, "Right SPO" },
+};
```

Comment:
This is exactly the same as `rt1011_dapm_routes[]` and I can expect that it will be the same when new Realtek amps are added. Can we remove `rt1011_dapm_routes[]` and keep this only?

Reply:
Ack, will update in next version.